apps/sms/views/shared/js/activity_handler.js

```diff
@@ -318,8 +318,9 @@ var ActivityHandler = {
             data: { id, threadId }
           };
 
-          return titlePromise.then((title) => {
-            var notification = new Notification(title, options);
+          return titlePromise.then((title) =>
+            NotificationHelper.send(title, options)
```

Comment:
Yay, cool! Btw, just looked into ```NotificationHelper.send``` - how do you think, should we set ```options.closeOnClick === false``` to keep our notification close-logic unchanged?